contracts/interfaces/USSSpokePoolInterface.sol

```diff
@@ -101,6 +123,19 @@ interface USSSpokePoolInterface {
         bytes updatedMessage
     );
 
+    event USSExecutedRelayerRefundRoot(
+        uint256 amountToReturn,
+        uint256 indexed chainId,
+        uint256[] refundAmounts,
+        uint32 indexed rootBundleId,
+        uint32 indexed leafId,
+        address l2TokenAddress,
+        address[] refundAddresses,
+        bytes32 fillsRefundedRoot,
+        string fillsRefundedIpfsHash,
+        address caller
```

Comment:
Same question as for the non-USS events - is the caller relevant if we assume Multicall3 continues to be used?

Reply:
Removed `caller` in this event but will keep in the original SpokePool event to reduce breaking changes in relayer-v2